app/src/components/layout/Header.vue

```diff
@@ -1,59 +1,164 @@
+                    class="my-2"
+                    color="grey"
+                    justify="left"
+                    rounded
+                    router
+                    text
+                    style="margin-top: 1vh"
+                    v-for="(link, index) in links">
+                {{ link.text }}
+            </v-btn>
 
-        <v-col  cols="1" md ="1" style="margin-top:1%; margin-right:1%">
-          <v-autocomplete
-            label="Umkreis"
-            :items="volunteerRadius"
-             v-model="selectedRadius"
-          >Überall</v-autocomplete>
-         </v-col> 
+            <v-spacer></v-spacer>
+            <v-col
+                    cols="4"
+                    md="4"
+                    style="background: #eee; border-radius: 20px">
+                <v-autocomplete
+                        :items="volunteerProposals"
+                        :search-input.sync="searchString"
+                        @change="tagChange"
+                        append-icon="search"
+                        clearable
+                        item-text="tag"
+                        style="margin-left: 10px; margin-right: 10px"
+                        v-model="searchResult">
+                </v-autocomplete>
+                <v-chip-group
+                        active-class="primary-text"
+                        column
+                        style="margin-left: 10px; margin-right: 10px; margin-top: -20px">
+                    <v-chip
+                            :key="tag"
+                            @click:close="remove(tag)"
+                            close
+                            v-for="tag in selectedTags">
+                        {{ tag }}
+                    </v-chip>
+                </v-chip-group>
+            </v-col>
+
+            <v-col
+                    cols="2"
+                    md="2">
+                <v-autocomplete
+                        :items="volunteerCities"
+                        label="Standort"
+                        prepend-inner-icon="place"
+                        style="margin-left: 10px; margin-right: 10px;"
+                        v-model="selectedCity">
+                    Mein Standort
+                </v-autocomplete>
+            </v-col>
+
+            <v-col cols="1" md="1">
+                <v-autocomplete
+                        :items="volunteerRadius"
+                        label="Umkreis"
+                        v-model="selectedRadius">
+                    Überall
+                </v-autocomplete>
+            </v-col>
+            <v-spacer></v-spacer>
+
+            <v-btn
+                    style="margin-top: 1vh"
+                    icon>
+                <v-icon>more_vert</v-icon>
+            </v-btn>
         </v-row>
-   <!--</v-container>-->
-  </v-header>
+    </header>
 </template>
 
-<script>
-export default {
-  data: () => ({
-    links: [
-      { text: 'Logo', route: '/' },
+<script lang="ts">
```

Comment:
This is very much duplicated code from Home.vue please use good software engeniering practices like DRY